static/common.js

```diff
@@ -1176,19 +1176,41 @@ function onUnitsOrInventoryLoaded() {
             });
 
         } else {
+            // Fix older versions/missing data
             for (var index in ownedUnits) {
                 if (ownedUnits[index] != "version" && typeof ownedUnits[index] === 'number') {
                     ownedUnits[index] = {"number":ownedUnits[index], "farmable":0};
                 }
             }
-            $("#inventoryDiv .status").text("loaded (" + Object.keys(itemInventory).length + " items, "+ Object.keys(ownedUnits).length + " units)");
+
+            updateUnitAndItemCount();
+
             $("#inventoryDiv .loader").addClass("hidden");
             $(".logOut").removeClass("hidden");
             inventoryLoaded();
         }
     }
 }
 
+function updateUnitAndItemCount() {
+    // Count units
+    var unitCount = 0;
+    Object.values(ownedUnits).forEach(unit => { unitCount += (unit.number || 0) + (unit.sevenStar || 0); });
```

Comment:
I needed to reword this line to change Object.values to use Object.keys. Some users with old firefox version were having problem. Object.values is not supported before Firefox 47, and the user typically had a version 45 (at work, cannot update)